contributors/guide/README.md

```diff
@@ -0,0 +1,212 @@
+**OWNER:**
+
+SIG-ContribX
+
+
+# WARNING
```

Comment:
Starting a new contributor document with an H1 WARNING isn't super friendly. Perhaps say this is a disclaimer?

Reply:
+1